dev-libs/libconfig/libconfig-9999.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+AUTOTOOLS_AUTORECONF="1"
+inherit eutils autotools-multilib
+
+DESCRIPTION="Libconfig is a simple library for manipulating structured configuration files"
+HOMEPAGE="http://www.hyperrealm.com/libconfig/libconfig.html"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/hyperrealm/libconfig.git"
+	EGIT_BRANCH="master"
+	inherit git-r3
+	KEYWORDS=""
+else
+	SRC_URI="http://www.hyperrealm.com/${PN}/${P}.tar.gz"
+	KEYWORDS="amd64 ~arm ~mips ppc ppc64 sparc x86 ~x86-linux"
+	PATCHES=( "${FILESDIR}/${P}-out-of-source-build.patch" )
+fi
+
+IUSE="+cxx examples static-libs"
+
+DEPEND="
+	sys-devel/libtool
+	sys-devel/bison"
```

Comment:
Possible.. I only modified the the ebuild from the gentoo main tree a little to build with the github source. Anyway, done :)